src/GeneralStateTestsFiller/stCreate2/CREATE2_EOF1InvalidFiller.yml

```diff
@@ -164,6 +164,143 @@ CREATE2_EOF1Invalid:
+      - ':yul { mstore(0, 0xef000101002600600160015560026002557b00f3fdfeff00f3fdfeff00f3fdfe) mstore(32, 0xff00f3fdfeff00f3fdfeff00f300000000000000000000000000000000000000)  return(0, 45) }'
+      # EOF Code: { sstore(0x1, 0x1) sstore(0x2, 0x2) push29( - ef000101000b00600160015560026002557c
+      - ':yul { mstore(0, 0xef000101000b00600160015560026002557c0000000000000000000000000000)  return(0, 18) }'
+      # EOF Code: { sstore(0x1, 0x1) sstore(0x2, 0x2) push29(0x00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fd - ef000101002700600160015560026002557c00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fd
+      - ':yul { mstore(0, 0xef000101002700600160015560026002557c00f3fdfeff00f3fdfeff00f3fdfe) mstore(32, 0xff00f3fdfeff00f3fdfeff00f3fd000000000000000000000000000000000000)  return(0, 46) }'
+      # EOF Code: { sstore(0x1, 0x1) sstore(0x2, 0x2) push30( - ef000101000b00600160015560026002557d
+      - ':yul { mstore(0, 0xef000101000b00600160015560026002557d0000000000000000000000000000)  return(0, 18) }'
+      # EOF Code: { sstore(0x1, 0x1) sstore(0x2, 0x2) push30(0x00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfe - ef000101002800600160015560026002557d00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfe
+      - ':yul { mstore(0, 0xef000101002800600160015560026002557d00f3fdfeff00f3fdfeff00f3fdfe) mstore(32, 0xff00f3fdfeff00f3fdfeff00f3fdfe0000000000000000000000000000000000)  return(0, 47) }'
+      # EOF Code: { sstore(0x1, 0x1) sstore(0x2, 0x2) push31( - ef000101000b00600160015560026002557e
+      - ':yul { mstore(0, 0xef000101000b00600160015560026002557e0000000000000000000000000000)  return(0, 18) }'
+      # EOF Code: { sstore(0x1, 0x1) sstore(0x2, 0x2) push31(0x00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff - ef000101002900600160015560026002557e00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff
+      - ':yul { mstore(0, 0xef000101002900600160015560026002557e00f3fdfeff00f3fdfeff00f3fdfe) mstore(32, 0xff00f3fdfeff00f3fdfeff00f3fdfeff00000000000000000000000000000000)  return(0, 48) }'
+      # EOF Code: { sstore(0x1, 0x1) sstore(0x2, 0x2) push32( - ef000101000b00600160015560026002557f
+      - ':yul { mstore(0, 0xef000101000b00600160015560026002557f0000000000000000000000000000)  return(0, 18) }'
+      # EOF Code: { sstore(0x1, 0x1) sstore(0x2, 0x2) push32(0x00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00 - ef000101002a00600160015560026002557f00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00
+      - ':yul { mstore(0, 0xef000101002a00600160015560026002557f00f3fdfeff00f3fdfeff00f3fdfe) mstore(32, 0xff00f3fdfeff00f3fdfeff00f3fdfeff00000000000000000000000000000000)  return(0, 49) }'
+
+      # EIP-3670: Valid legacy initcode returning invalid EOF code - Containing undefined instruction
```

Comment:
Do we still need this here if there's another separate file for invalid opcodes deployment?